chidioguejiofor/airtech-api
# #164838806: Adds setup for docker

#### What does this PR achieve?
- Adds Dockerfile to the app
- Adds config for docker-compose via docker-compose.yml
- Adds entry point script for docker
- Sets up docker compose in the app
- Adds  .env-sample to the app 

#### Background Context?
- Docker would be used to host the app once it is time

#### New environmental Variables?
- None

#### Manual Testing Steps
- Download and install docker from [here](https://www.docker.com/get-started)
- Open the project directory in your terminal
- Run `docker-compose build` to build the image from  `docker/Dockerfile`
- Run `docker-compose up` to start the app
- Hit the endpoint `localhost` in your app
- You should receive the default route request below:

```
{
    "message": "The resource you specified does not exist"
}
```

#### Screenshots:
![Screenshot 2019-04-04 at 3 38 54 PM](https://user-images.githubusercontent.com/38565349/55564452-d4456180-56ef-11e9-98bc-b050ccdb7659.png)

#### Any new deployment instructions
- None
